### PR TITLE
MM-42571: Pre-package Playbooks v1.26.0

### PR DESCRIPTION
**Pull request by Phrynobatrachus**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-42571

#### Release Note
```release-note
Pre-package Playbooks v1.26.0

Playbooks specific release notes can be found at:
https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.26.0
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -667,7 +667,7 @@ gen-serialized: ## Generates serialization methods for hot structs
 	@mv tmp.go ./model/team_member_serial_gen.go
 
 todo: ## Display TODO and FIXME items in the source code.
-	@! ag --ignore Makefile --ignore-dir vendor --ignore-dir runtime '(TODO|XXX|FIXME|"FIX ME")[: ]+' 
+	@! ag --ignore Makefile --ignore-dir vendor --ignore-dir runtime '(TODO|XXX|FIXME|"FIX ME")[: ]+'
```
Comment (lieut-data) [inline Makefile:670]: nit: can we omit this change from this PR? Just so that we don't mix linting issues with pre-packaged changes.

**Review by lieut-data — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Phrynobatrachus) [reply Makefile]: whoops, done

**Comment by lieut-data** [comment]
Looks good, thanks @Phrynobatrachus!

**Comment by lieut-data** [comment]
I think we can ignore this error, since we just managed to beat cutting the release branch this time, and it should get included organically. :tada:

**Comment by lieut-data** [comment]
Ah, thank you @amyblais!

**Pull request merged**